README.md

```diff
@@ -26,28 +26,45 @@
      ```
      sudo apt-get install python3
      ```
-  2. clone the master branch
-  3. create a virtual environment in the project root directory
-  4. activate the virtual environment ```source venv/bin/activate```
-  5. pip install requiremnets ```pip install -r requirements.txt```
-  6. create a sqlite database ```touch dev.db```
-  7. set dev database environment variable ```export DEV_DATABASE_URL=sqlite:///dev.db```
-  8. Open the database in sqlite with ```sqlite3 dev.db;``` check to see if it created the tables with ```.tables```
-  9. try to display data from a table ```select * from projects;``` you should see a list of projects display
+  2. [Install virtualenv](https://virtualenv.pypa.io/en/latest/installation/)
+     ```
+     sudo apt install virtualenv
+  3. clone the master branch
+     ```
+     git clone git@github.com:codeforkansascity/meep-backend.git
+  4. move into project root directory
+     ```
+     cd meep-backend
+  5. create a virtual environment in the project root directory
+     ```
+     virtualenv venv
```

Comment:
I normally use venv and create the virtual environment with ```python3 -m venv venv```. Is this equivalent?

Reply:
Oh, okay. I saw step 3 on the windows installation, which said install virtualenv, so I went with that.

There's another tool specifically called virtualenv.
https://virtualenv.pypa.io/en/stable/

I didn't realize that venv was a module for creating virtual environments. I thought that venv was just the name you chose for your virtual environment. virtualenv shows ENV in their documentation.
https://virtualenv.pypa.io/en/stable/userguide/

I just checked my system at home, it had virtualenv already installed.
I also checked python3, and it has venv and virtualenv preinstalled.

It looks like either way would work.

either
```virtualenv /path/to/new/virtual/environment```
or
```python3 -m virtualenv /path/to/new/virtual/environment``
or
```python3 -m venv /path/to/new/virtual/environment```

Can you check your python3 to see if it has the virtualenv option? I might lean towards the second or third option, because it eliminates the need to install virtualenv if your distribution didn't include it by default.

Also, do you want to make a call on what to designate as the name of the virtual environment.

Here's a comparison of the different virtual environment options.
https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe

Reply:
@aaronshivers thanks for the overview. I think popular virtual environment names are env, venv, and the name of the project. Since venvs shouldn't be tracked in version control, why don't we throw all three in .gitignore and leave it up to the developer.

Reply:
As for the venv vs. virtualenv question, I don't think it matters too much. The venv module has been part of the python standard library since python 3.3, and we're running 3.7, so I believe either should work.

Reply:
There's a pending PR to build and run the app in a docker container, which will eliminate the need for a virtual environment.